cranelift-codegen/Cargo.toml

```diff
@@ -58,6 +59,9 @@ arm32 = []
 arm64 = []
 riscv = []
 
+# For dependent crates that want to serialize some parts of cranelift
+cranelift-serde = ["serde"]
```

Comment:
Would you mind renaming this to `enable-serde`, so that it isn't confused with the `cranelift-serde` crate?

Reply:
Will do.